apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts

```diff
@@ -82,7 +83,7 @@ router.get(
       'success',
       'Question copied successfully. You are now viewing your copy of the question.',
     );
-    res.redirect(res.locals.urlPrefix + '/question/' + question.id);
+    res.redirect(`${res.locals.urlPrefix}/question/${question.id}/settings`);
```

Comment:
This is the core change in functionality, everything else is just a conversion to TS.